GVFS/GVFS/RepairJobs/GitRefsHeadsRepairJob.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using GVFS.Common;
+using GVFS.Common.Tracing;
+
+namespace GVFS.RepairJobs
+{
+    public class GitRefsHeadsRepairJob : GitRefsRepairJob
+    {
+        public GitRefsHeadsRepairJob(ITracer tracer, TextWriter output, GVFSEnlistment enlistment)
+            : base(tracer, output, enlistment)
+        {
+        }
+
+        public override string Name
+        {
+            get { return @".git\refs\heads\**\*"; }

```

Comment:
This isn't what your code is actually doing. Your code is checking for anything in `.git\refs\heads\*`

Reply:
I guess it depends on what wildcard syntax you're using. I read this as `.git\refs\heads\<some subfolder here>\*`. Either way, this is just a display string and intended to inform users, so lets remove the ambiguity. You can just say `Local branches` here and be totally clear about what you mean.